views/notification.ejs

```diff
@@ -0,0 +1,6 @@
+<div id='toast-wrapper' aria-live="polite" aria-atomic="true" style="position: relative; z-index: 999;">
+  <div class="toast" style="position: absolute; top: 0; right: 0;">
+    <div class="toast-header"></div>
```

Comment:
Can you shift all this styling to `style.css`. And some more style changes

- give some background color to toast, now it matches the background, which doesn't looks good.

- at present this toast is at extreme top and right give it some padding.

- put a cancel button on every toast.

- give header and body different shades so that they are readable.

- when multiple notification need to be shown, stack toasts over one another, now previous notification just collapses(hides).